cfme/fixtures/v2v.py

```diff
@@ -20,15 +21,23 @@ def v2v_providers(request, second_provider, provider):
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
     """Add credentials to conversation host"""
```

Comment:
I think the doc block here needs to be updated to better explain what is happening in this fixture.